OpenRA.Mods.D2k/Traits/SpiceBloom.cs

```diff
@@ -41,12 +42,15 @@ public class SpiceBloomInfo : TraitInfo, IRenderActorPreviewSpritesInfo, Require
 		[WeaponReference]
 		public readonly string Weapon = null;
 
-		[Desc("The amount of spice to expel.")]
-		public readonly int[] Pieces = { 2, 12 };
+		[Desc("The amount of spice to expel. First number represent minimum amount. Second Maximum expel when actor is fully grow up")]
```

Comment:
Maybe something like `The number of times to fire Weapon at the minimum and maximum actor age`?